Mathlib/Data/Set/Intervals/Basic.lean

```diff
@@ -756,6 +756,10 @@ theorem Icc_self (a : α) : Icc a a = {a} :=
   Set.ext <| by simp [Icc, le_antisymm_iff, and_comm]
 #align set.Icc_self Set.Icc_self
 
+instance instIccUnique : Unique (Set.Icc a a) where
+  default := ⟨a, by simp⟩
+  uniq y := Subtype.ext <| by simpa using y.2
+
```

Comment:
This came from merging master into this branch. Hopefully that doesn't mess up the version history.